model/configuration.py

```diff
@@ -769,3 +774,567 @@ def excluded_audio_data_sources(cls, _db):
         if value is None:
             value = cls.EXCLUDED_AUDIO_DATA_SOURCES_DEFAULT
         return value
+
+
+class ExternalIntegrationOwner(object):
```

Comment:
"Owner" sounds like a business-level relationship, the way a library "owns" a license to a book. Since this is a mixin class I recommend calling it `HasExternalIntegration`, similar to `HasSelfTests` in `selftest.py`.

Reply:
Changed it to `HasExternalIntegration`